### DISCUSSION TITLE
Utilizzare uno switchbot come interruttore per accendere e spegnere il PC

### DISCUSSION THREAD

**Discussion by luxbarba** [indomus/forum#1633] [HUB: Home Assistant]
Buongiorno a tutti. Ho acquistato uno switchbot come interruttore per accendere e spegnere il pc. Sono già riuscito a far riconoscere lo Switchbot ad home assistant. Per spegnere il PC prima utilizzavo l'integrazione rpc_shutdown tramite la piattaforma wake on lan. Ho cercato di riutilizzare lo stesso script che avevo in precedenza con rpc shutdown ma non mi funziona lo script. Allego qui lo script oscurando il mac address del pc.
  Volevo continuara ad utilizzare questo tipo di codice in quanto mi determina lo stato del pc. 
```switch: 
  - platform: wake_on_lan
    mac: xx:xx:xx:xx:xx:xx
    name: "Pc-Studio"
    host: 192.168.1.4
    turn_off:
      service: switch.turn_off
      data:
        addon: SwitchBot
        input: Interruttore PC
    turn_on:
      service: switch.turn_on
      data:
        addon: SwitchBot
        input: Interruttore PC```

**Comment by 50494554524F** [comment]
lo switchbot è bluetooth, usa switchbot_press di hacs...

**Reply by luxbarba** [reply]
Grazie mille!!!!
